src/main/java/org/apache/datasketches/Util.java

```diff
@@ -452,14 +453,36 @@ public static int ceilingPowerOf2(final int n) {
    * integer.</li>
    * </ul>
    *
-   * @param n The given argument.
-   * @return the floor power of 2.
+   * @param n The given int argument.
+   * @return the floor power of 2 as an int.
    */
   public static int floorPowerOf2(final int n) {
     if (n <= 1) { return 1; }
     return Integer.highestOneBit(n);
   }
 
+  /**
+   * Computes the floor power of 2 given <i>n</i> is in therange [1, 2^63-1].
+   * This is the largest positive power of 2 that equal to or less than the given n and equal
+   * to a mathematical integer.
```

Comment:
There is a slight typo in the javadoc for this and the previous method:
```This is the largest positive power of 2 that _is_ equal```

Reply:
Thanks fixed.